packages/model-viewer/src/features/scene-graph/texture-info.ts

```diff
@@ -157,6 +163,24 @@ export class TextureInfo implements TextureInfoInterface {
           case TextureUsage.ClearcoatNormal:
             material.clearcoatNormalMap = threeTexture;
             break;
+          case TextureUsage.SheenColor:
```

Comment:
I don't love this huge switch statement - if you can think of a cleaner way to architect this, I'm open to it, but only if you're feeling inspired.